### ISSUE TITLE
a question of Children pass-through

### ISSUE THREAD

**Issue by yueshuiniao** [chantastic/reactpatterns#34]
```js
// option 1: extra div
return <div>{children}</div>

// option 2: unhelpful errors
return children
```
I try to return this.props.children directly，and there is no error. I test the version 15.4 and 16, Here is the [demo](https://codepen.io/yueshuiniao/pen/BmmdzY).

So Is there something I misunderstood?

**Comment by newyork-anthonyng** [comment]
When trying to return multiple children and using `React.Children.only`, you get this error message:

> React.Children.only expected to receive a single React element child.

I'm wondering if this is the helpful error that is being mentioned? It is unclear to me too.

**Comment by vardhanreddy27** [comment]
Yes, the error message "React.Children.only expected to receive a single React element child" is the one that is displayed when using React.Children.only and attempting to return multiple children.

React.Children.only is a utility function that is used to ensure that a component only has a single child element. It is often used when a component expects to receive a single child element, but could potentially receive multiple children if the parent component is not written correctly.

For example, consider the following component:
function MyComponent(props) {
  return (
    <div>
      {props.children}
    </div>
  );
}
This component expects to receive a single child element, but it could potentially receive multiple children if the parent component passes them in. For example:
<MyComponent>
  <div>Child 1</div>
  <div>Child 2</div>
</MyComponent>
In this case, the MyComponent would receive two children, which is not what it was designed to handle. To prevent this situation, you can use React.Children.only to ensure that the component only receives a single child:
function MyComponent(props) {
  const child = React.Children.only(props.children);
  return (
    <div>
      {child}
    </div>
  );
}
If the MyComponent is passed multiple children, it will display the error message "React.Children.only expected to receive a single React element child."

**Comment by vardhanreddy27** [comment]
Yes, the error message "React.Children.only expected to receive a single React element child" is the one that is displayed when using React.Children.only and attempting to return multiple children.

React.Children.only is a utility function that is used to ensure that a component only has a single child element. It is often used when a component expects to receive a single child element, but could potentially receive multiple children if the parent component is not written correctly.
<img width="212" alt="image" src="https://user-images.githubusercontent.com/63119222/210737920-1c6d4926-a081-4f82-99f3-0d0c118c8545.png">
This component expects to receive a single child element, but it could potentially receive multiple children if the parent component passes them in. For example:
<img width="143" alt="image" src="https://user-images.githubusercontent.com/63119222/210738030-b3265237-ad2a-4a39-bff8-44e1f05e5c3d.png">
In this case, the MyComponent would receive two children, which is not what it was designed to handle. To prevent this situation, you can use React.Children.only to ensure that the component only receives a single child:
<img width="355" alt="image" src="https://user-images.githubusercontent.com/63119222/210738140-4bf93c5b-c826-4ff0-b10d-35925a552877.png">
If the MyComponent is passed multiple children, it will display the error message "React.Children.only expected to receive a single React element child."
